Tests/NooraTests/Components/SingleChoicePromptTests.swift

```diff
@@ -34,7 +33,7 @@ struct SingleChoicePromptTests {
         keyStrokeListener.keyPressStub = [.downArrowKey, .upArrowKey]
 
         // When
-        _ = subject.run()
+        let _: Option = subject.run()
```

Comment:
is this needed?

Reply:
Yes. Otherwise, it can't resolve the generic type.